OpenRobertaRobot/src/main/java/de/fhg/iais/roberta/factory/RobotFactory.java

```diff
@@ -64,7 +64,7 @@ public final String getBinaryFileExtension() {
 
     @Override
     public final String getGroup() {
-        String group = this.pluginProperties.getStringProperty("robot.plugin.group");
+        final String group = this.pluginProperties.getStringProperty("robot.plugin.group");
```

Comment:
Check save actions, I believe the only one for final is `Add final modifier to private fields`.